spec/system/task_spec.rb

```diff
@@ -1,24 +1,25 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { FactoryBot.create(:project) }
+
   describe 'Task一覧' do
+    let(:task) { FactoryBot.create(:task, project: project) }
```

Comment:
Factoryにassociationを記載すると、projectを指定しなくてもテストデータが作成できるようになります。
```suggestion
    let(:task) { create(:task) }
```

Reply:
1034dbf